src/WizardSteps/components/WizardNavigations/WizardNavigations.tsx

```diff
@@ -74,15 +80,41 @@ const WizardNavigations: React.FC<WizardNavigationsProps> = ({
         </div>
       </div>
       {
-        <CSSTransition classNames="list-group" in={toggle} timeout={400}>
-          <ol className="list-group list-group-ordered d-md-flex mt-3">
+        <CSSTransition
+          nodeRef={olRef}
+          classNames="list-group"
+          in={toggle}
+          timeout={400}
+        >
+          <ol
+            ref={olRef}
+            className="list-group list-group-ordered d-md-flex mt-3"
+          >
             {navigations.map((props: WizardNavigationData, step: number) => {
               return (
                 <WizardNavigation
                   key={`${props.path}_${props.label}`}
                   {...props}
                   step={step}
-                  onClick={() => isValidStep(step) && setToggle(false)}
+                  onClick={(event) => {
+                    if (isNavigableStep(step)) {
+                      const isForwardNavigation = step > activeStep;
+
+                      if (isForwardNavigation) {
+                        event.preventDefault();
```

Comment:
maybe can add an explanation for calling prevent default here? so others will understand the reason for this line and line 115?

Reply:
added